joomla/joomla-cms
# [3.9] Make sure we don't send a referrer on the remind and reset views

### Summary of Changes

Make sure we don't send a referrer on the remind and reset views

### Testing Instructions

1) Apply this patch
2) go to the remind page and request and remind token
3) click on the link that is send to you
4) check the http header of that page with the F12 developer tools.

### Expected result

On this specific page we set the Referrer-Policy and set it to no-referrer so no referrer is send.

### Actual result

We do not send that policy.

### Documentation Changes Required

None. 

### Additional Information

This is a pull request for an report to the security team where the @joomla/security team decided to move the patch to the public tracker.

Thanks to Pim Feltkamp & Vipul Sahu for reporting this to us.
